blog/_posts/2020-11-06-hybrid-cloud-with-inlets.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: post
+title: Hybrid Cloud with inlets
+description: Learn how to connect on-premises services and cloud services with inlets &reg;
+author: Johan Siebens
+tags: inlets-pro hybrid-cloud 
+author_img: jsiebens
+date: 06-11-2020
+---
+
+Learn how to connect on-premises services and cloud services with inlets &reg;
```

Comment:
This could be marginally longer to mention that it uses Wordpress as an example. You can preview how it looks on the blog rollout http://localhost:4000/blog/